pkg/ebpf/c/tracee.bpf.c

```diff
@@ -795,6 +795,11 @@ static __always_inline u32 get_task_mnt_ns_id(struct task_struct *task)
     return get_mnt_ns_id(READ_KERN(task->nsproxy));
 }
 
+static __always_inline u32 get_task_nsproxy_pid_ns_id(struct task_struct *task)
+{
+    return get_pid_ns_id(READ_KERN(task->nsproxy));
```

Comment:
This function should probably be renamed as well to `get_pid_ns_for_children_id`

Reply:
So what I actually meant was to rename the two functions:
`get_task_pid_ns_for_children_id` (the outer function youv'e added) and `get_pid_ns_for_children_id`  (this function call)

Reply:
Sorry to misunderstand, will fix it soon.

Reply:
> So what I actually meant was to rename the two functions:
> 
> `get_task_pid_ns_for_children_id` (the outer function youv'e added) and `get_pid_ns_for_children_id`  (this function call)

Sorry to misunderstand, will fix it soon.

Reply:
> So what I actually meant was to rename the two functions: `get_task_pid_ns_for_children_id` (the outer function youv'e added) and `get_pid_ns_for_children_id` (this function call)

it has been fixed, please help to have a look when you are convinent, tks.